sbin/pack.sh

```diff
@@ -70,7 +72,14 @@ OSNICK=$($READIES/bin/platform --osnick)
 [[ $OSNICK == rocky8 ]]  && OSNICK=rhel8
 [[ $OSNICK == rocky9 ]]  && OSNICK=rhel9
 
-[[ $OSNICK == bigsur ]]  && OSNICK=catalina
+if [[ $OS == macos ]]; then
+	# as we don't build on macOS for every platform, we converge to a least common denominator
+	if [[ $ARCH == x86_64 ]]; then
+		OSNICK=catalina  # to be aligned with the rest of the modules in redis stack
+	else
+		[[ $OSNICK == ventura ]] && OSNICK=monterey
```

Comment:
Not familiar with such if syntax, can we use elif?

Reply:
I copied it from other modules: https://github.com/RediSearch/RediSearch/blob/4a077d8de97e1c9ebcf0e265da3802c5793b0c1c/sbin/pack.sh#L84

Looks like they fixed it for other modules but forgot bloom and timeseries. 
I agree it is really weird but I think we should try to keep it same across modules for now. So, next time it'd be easier to see the diff in these files.

Reply:
Let the chaos continue!